README.md

```diff
@@ -175,6 +175,8 @@ Configuration format is described in detail in the
 * `cartridge_enable_tarantool_repo` (`boolean`, optional, default: `true`):
   indicates if the Tarantool repository should be enabled (for packages with
   open-source Tarantool dependency);
+* `cartridge_remove_temp_files` (`boolean`, optional, default: `false`):
```

Comment:
What is temp files for user? Maybe `cartridge_cleanup_package_file`?

BTW, there is one more tmp file we need to remove - repository setup script.

Reply:
The script has already been removed.

Temporary files are temporary files created by a role, but these files are not needed by the application.